lib/firecode_check_bb_impl.cc

```diff
@@ -80,14 +80,14 @@ namespace gr {
 
       while (d_nconsumed < noutput_items / d_frame_size - 4) {
         if (fc.check(&in[d_nconsumed * d_frame_size])) {
-          GR_LOG_DEBUG(d_logger, format("fire code (%d %d) OK at frame %d") %(int)in[d_nconsumed*d_frame_size] %(int)in[d_nconsumed*d_frame_size+1] % (nitems_read(0) / d_frame_size));
+          //GR_LOG_DEBUG(d_logger, format("fire code (%d %d) OK at frame %d") %(int)in[d_nconsumed*d_frame_size] %(int)in[d_nconsumed*d_frame_size+1] % (nitems_read(0) / d_frame_size));
```

Comment:
GR_LOG_DEBUG should "comment itself away" if not compiled with debug flags.

Hence, either delete or uncomment such lines.

Generally, I'm personally in favor of heavy debug logging, but that's a choice that you should absolutely do on your own.